--- a/net/wireless/Kconfig
+++ b/net/wireless/Kconfig
@@ -89,7 +89,8 @@ config CFG80211_CERTIFICATION_ONUS
        help
          ...

 config CFG80211_REQUIRE_SIGNED_REGDB
-       bool "require regdb signature" if CFG80211_CERTIFICATION_ONUS
+       bool "require regdb signature" if CFG80211_CERTIFICATION_ONUS
+       depends on CFG80211_CERTIFICATION_ONUS
        default y
        help
          ...
